src/wagtail_ai/migrations/0002_populate_default_prompts.py

```diff
@@ -0,0 +1,31 @@
+# Created using Django 4.2.7 on 2023-12-06 10:07
+
+from django.db import migrations
+
+from wagtail_ai.models import DEFAULT_PROMPTS
+
+
+def set_default_ai_prompts(apps, schema_editor):
+    """
+    Loop through each default prompt and
+    populate the database.
+    """
+    Prompt = apps.get_model("wagtail_ai", "Prompt")
+    for default_prompt in DEFAULT_PROMPTS:
+        Prompt.objects.update_or_create(
+            uuid=default_prompt["uuid"],
+            label=default_prompt["label"],
+            prompt=None,  # Left blank to allow users to override the prompt value and maintainers of wagtail AI to manage the prompt in the codebase.
+            description=default_prompt.get("description", ""),
+            method=default_prompt.get("method", None),
+        )
```

Comment:
Sorry not sure what you mean here, could you expand

I could update the code to the following?
```
    Prompt = apps.get_model("wagtail_ai", "Prompt")
    for default_prompt in DEFAULT_PROMPTS:
        default_prompt.pop("prompt", None)
        Prompt.objects.update_or_create(
            **default_prompt,
            prompt=None,  # Left blank to allow users to override the prompt value and maintainers of wagtail AI to manage the prompt in the codebase.
        )
```

Reply:
I thought we only want to update fields that match a specific criteria I guess, i.e. uuid match. So you only lookup the object by UUID, rather than having a long WHERE clause. I may be misunderstanding what we're trying to do here though.

See the docs: https://docs.djangoproject.com/en/stable/ref/models/querysets/#update-or-create

Otherwise, if the prompt is not set to None or some other field has been changed, you will be creating a new object rather than updating the old one, I guess.

```python
Prompt.objects.update_or_create(
    uuid=default_prompt["uuid"],
    defaults={
        "label": default_prompt["label"],
        "prompt": None,  # Left blank to allow users to override the prompt value and maintainers of wagtail AI to manage the prompt in the codebase.
        "description": default_prompt.get("description", ""),
        "method": default_prompt.get("method", None),
    }
}
```

Reply:
That makes, sense, thanks for taking the time to explain this!